packages/commonjs/src/index.js

```diff
@@ -88,7 +88,9 @@ export default function commonjs(options = {}) {
         ? options.ignoreTryCatch(id)
         : Array.isArray(options.ignoreTryCatch)
         ? options.ignoreTryCatch.includes(id)
-        : options.ignoreTryCatch || false;
```

Comment:
```ts
options.ignoreTryCatch ?? true
```

is a bit shorter if we can use that operator

Reply:
I think it's only supported in Node 14 and newer? (I'm not sure the name of the operator, so not quite sure how to Google it to check). Are there certain versions of Node that we support? I think I'd probably just leave it as is for now to keep supporting Node 12

Reply:
Yes, as this plugin does not have a Babel or TypeScript step (yet), we need to make sure it runs in Node 12 unfortunately.